orchestrator/test_runner/src/happy_path.rs

```diff
@@ -443,6 +418,53 @@ pub async fn test_erc20_deposit_result(
     ))
 }
 
+pub async fn send_erc20_deposit(
+    web30: &Web3,
```

Comment:
this looks famiilar so I'm sure it's been duplicated with slight modifications from somewhere, it doesn't seem to have been moved. Can it be de-duplciated?

Reply:
I pulled this segment out of the function above, that function now calls this to avoid duplication.